cmd/getEnvironment.go

```diff
@@ -99,6 +99,8 @@ func NewGetEnvironmentCommand() *cobra.Command {
 			case "azure":
 				items = append(items, Item{"SUBSCRIPTION_ID", env.ProviderOptions.Azure.SubscriptionID})
 				items = append(items, Item{"APPLICATION_ID", env.ProviderOptions.Azure.ApplicationID})
+			case "google":
+				items = append(items, Item{"PROJECT_ID", env.ProviderOptions.Google.ProjectID})
```

Comment:
Should this display Service Account Email as well?

Reply:
I can print it out but I see the email as "confidential"...